contrib/cni/README.md

```diff
@@ -1,16 +0,0 @@
-## `contrib/cni` ##
```

Comment:
Do we need to remove this? It seems like a helpful pointer to where to get the plugins, if nothing else.

Reply:
imho, they should be installing a containersnetworking-cni package.  we have to get away from catering to installation from source.